dvc/command/metrics.py

```diff
@@ -159,7 +159,7 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    METRICS_HELP = "Commands to add, manage, collect and display metrics."
+    METRICS_HELP = "Commands to add, manage, collect, and display metrics."
```

Comment:
Are you sure comma is needed here? `and` is not really joining two independent clauses.

Reply:
There's never absolute correctness with these things, but yes, in general everyone recommends to add a comma before "and/or" in the last element of a list. See https://lifehacker.com/always-put-a-comma-before-and-in-a-list-no-matter-wh-1793376109